### ISSUE TITLE
Documentation unclear

### ISSUE THREAD

**Issue by ssddOnTop** [est31/rcgen#104]
Documentation unclear, how to use the public/private keys for encryption and decryption

**Comment by ssddOnTop** [comment]
Yes, Im talking about RSA, is there anyway that we send RSA public key inside thr x509 cert, it might be some custom value like ‘e: 65537’, ‘n: p*q’ and we can send that cert? So that client can use that for encryption directly 

And sorry but the docs for x509 parsar is unclear as well, Im too confused.

**Comment by est31** [comment]
@ssddOnTop you can do this all with the RSA crate: https://docs.rs/rsa/latest/rsa/struct.RsaPrivateKey.html

**Comment by est31** [comment]
(Whether I would recommend it is another question, I wouldn't)
